src/views/guidelines/guidelines.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../colors";
+@import "../../typography";
+
+.view {
+    .guidelines-footer {
+        text-align: center;
```

Comment:
@TheGrits this isn't aligning center for me when I look at it in my development environment right now – is it for you?